### PR TITLE
feat: detect when Guarantee chat should be displayed

### LINKED ISSUE #667: Display chat link for guarantee cases.
As a customer flying with Kiwi.com, I want to be able to get in touch with our support.

The link should be only accessible when departure is within next 4 hours & only when the upcoming flight is covered by Kiwi.com Guarantee.

The backend will be provided by Michal Knazsky and his team. Our main goal is tho create a new "threshold" based on the rules above on the backend and display the chat button.

I will add more detailed description shortly.

Designs:

Desktop:
gt chat

Mobile:
m1-3-0 help normal copy 3

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
closes #667<br/><br/><br/><url>LiveURL: https://smartfaq-enable-chat-button.surge.sh</url>

### CODE REVIEW

**Comment by jalbertsr** [comment]
If I understood the issue correctly the chat should only be shown if there are less than 4 hours for the departure, but I can see this: 

screenshot

File: src/SingleBookingPage/BookingDetail.js [hunk 1]
```diff
@@ -159,7 +159,9 @@ class BookingDetail extends React.Component<Props> {
         dataCy="nearestBooking"
         styles="width: 100%; padding:40px; background-color: #f5f7f9;"
       >
-        <Header booking={booking} isFuture={arrivalInfo.isFuture} />
+        <div className={arrivalInfo.isFuture ? 'upcoming' : 'past'}>
```
Comment (jalbertsr) [inline src/SingleBookingPage/BookingDetail.js]: I don't know if creating classes that are not used in CSS, but using it for testing is the best practice/approach.

File: .env.sample [hunk 2]
```diff
@@ -5,3 +5,7 @@ GRAPHQL_URI=
 GH_ACCESS_TOKEN=
 TEST_USER_EMAIL=
 TEST_USER_PASSWORD=
+CHAT_ORG_ID=
+CHAT_QUEUE_NAME=
+CHAT_GUID=
+CHAT_DEPLOYMENT_KEY=
```
Comment (jalbertsr) [inline .env.sample]: Will you provide these keys in 1password or via slack?

File: src/SingleBookingPage/BookingDetail.js (hunk 1 above)
Comment (jaroslav-kubicek) [reply src/SingleBookingPage/BookingDetail.js]: fixed

**Pull request merged**